### PR TITLE
Display AAB test certificate after first upload

### PR DESCRIPTION
**Pull request by tagboola**
### Description

Display test certificate after first AAB upload.

### Scenarios Tested

Tested locally by creating a new app and uploading an AAB. 
Screen Shot 2021-02-23 at 7 52 22 PM

### CODE REVIEW

**Review by pranavrajgopal — APPROVED** [review]

File: src/appdistribution/client.ts [hunk 1]
```diff
@@ -54,11 +67,7 @@ export class AppDistributionClient {
 
   constructor(private readonly appId: string) {}
 
-  async getApp(
-    distributionFileType: DistributionFileType = DistributionFileType.APK
-  ): Promise<AppDistributionApp> {
-    utils.logBullet(`getting app details (Distribution type: ${distributionFileType})...`);
-    const appView = distributionFileType == DistributionFileType.AAB ? "FULL" : "BASIC";
+  async getApp(appView: AppView = AppView.BASIC): Promise<AppDistributionApp> {
```
Comment (bkendall) [inline src/appdistribution/client.ts:70]: I _think_ the type here is redundant, but please feel free to ignore if it's required.

```suggestion
  async getApp(appView = AppView.BASIC): Promise<AppDistributionApp> {
```

**Review by bkendall — APPROVED** [review]
LGTM. Do you want to include a CHANGELOG.md entry?

File: src/appdistribution/client.ts (hunk 1 above)
Comment (tagboola) [reply src/appdistribution/client.ts]: Nice! Updated

**Comment by tagboola** [comment]
Good question! We actually don't want to add an entry in CHANGELOG.md as this feature is still in alpha

**Pull request merged**